src/map/script_constants.hpp

```diff
@@ -526,6 +526,8 @@
 	export_parameter(KAFRAPOINT_VAR, SP_KAFRAPOINTS);
 	export_parameter(PCDIECOUNTER_VAR, SP_PCDIECOUNTER);
 	export_parameter(COOKMASTERY_VAR, SP_COOKMASTERY);
+	export_parameter("LapineUpgradeItem", SP_LAST_LAPINE_UPGRADE_ITEM);
```

Comment:
will remove these 2, just use player temporary var

Reply:
If I take that out, should it work?